app.js

```diff
@@ -506,6 +512,12 @@ d.run(() => {
 				 * @todo Add description for the function and its params
 				 */
 				function(scope, cb) {
+					if (!appConfig.peers.enabled) {
+						scope.logger.info(
+							'Socket Cluster was NOT intiatied due to peers layer is disabled.'
```

Comment:
Skipping P2P server initialization due to the config settings - "peers.enabled" is set to false.
```suggestion
							'Socket Cluster was NOT intiatied due to peers layer is disabled.'
```